atomic_reactor/utils/cachi2.py

```diff
@@ -9,10 +9,54 @@
 """
 
 from typing import Any, Callable, Dict, Optional, Tuple, List
+from pathlib import Path
+import os.path
 
 from packageurl import PackageURL
 
 
+def _validate_paths(remote_sources_packages: dict) -> None:
+    """Paths must be relative and within cloned repo"""
+    def is_path_ok(path_string):
+        path = Path(path_string)
+        if path.is_absolute():
+            return False
+
+        fake_root = Path("/fake_root")
+        full_path = (fake_root/path).resolve()
```

Comment:
A path may be OK within a fake root but not within a real root (if it's a symlink in the real root)

Reply:
but I can rework it to use real cloned data, not a big issue to be sure that validation is complete

Reply:
Hmm, ok

Reply:
But yeah, if it's not a big change to run on the real repo, that would be nicer